[rospy.client][INFO] 2021-09-18 20:34:15,810: init_node, name[/server_endpoint], pid[26556]
[xmlrpc][INFO] 2021-09-18 20:34:15,811: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-09-18 20:34:15,811: Started XML-RPC server [http://wng:46247/]
[rospy.impl.masterslave][INFO] 2021-09-18 20:34:15,813: _ready: http://wng:46247/
[rospy.init][INFO] 2021-09-18 20:34:15,811: ROS Slave URI: [http://wng:46247/]
[rospy.registration][INFO] 2021-09-18 20:34:15,817: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-09-18 20:34:15,818: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-09-18 20:34:15,913: registered with master
[rospy.rosout][INFO] 2021-09-18 20:34:15,914: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-18 20:34:15,915: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-18 20:34:15,915: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-18 20:34:15,920: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-18 20:34:16,155: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-18 20:35:03,375: Connection from 192.168.1.146
[rosout][INFO] 2021-09-18 20:35:03,379: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-18 20:35:03,460: topic[/execution_check] adding connection to [http://wng:46247/], count 0
[rospy.internal][INFO] 2021-09-18 20:35:03,461: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rosout][ERROR] 2021-09-18 20:58:06,634: Exception: No more data available
[rosout][INFO] 2021-09-18 20:58:06,636: Disconnected from 192.168.1.146
[rospy.core][INFO] 2021-09-18 20:58:09,654: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-09-18 20:58:09,656: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][ERROR] 2021-09-18 20:58:09,657: Unable to initiate TCP/IP socket to wng:46109 (http://wng:46247/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-09-18 20:58:09,657: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-09-18 20:58:09,657: topic[/execution_check] removing connection to http://wng:46247/
[rospy.impl.masterslave][INFO] 2021-09-18 20:58:09,658: signal-2
[rospy.core][INFO] 2021-09-18 20:58:09,759: signal_shutdown [atexit]
